zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/migration/to_1_1/MessageSubscriptionSentTimeMigrationTest.java

```diff
@@ -37,11 +38,13 @@ public class MockBasedTests {
     @Test
     public void noMigrationNeededWhenColumnIsEmpty() {
       // given
-      final var mockProcessingState = mock(ProcessingState.class);
+      final var mockProcessingState = mock(MutableProcessingState.class);
```

Comment:
:upside_down_face: These are for older versions. Can be removed, no?

Reply:
I've asked the team here: https://camunda.slack.com/archives/C037W9NMATG/p1725346093198389